src/index.js

```diff
@@ -18,4 +19,9 @@ class Keypress extends PureComponent {
     }
 }
 
+Keypress.propTypes = {
+    onKeys: PropTypes.array,
+    onKeyPress: PropTypes.func,
+}
```

Comment:
Both of these props are required, can you add `isRequired` to both and the onKeys can have more details using the arrayOf type:
```
onKeys: PropTypes.arrayOf(PropTypes.number)
```